html/docs/include/features/install.html

```diff
@@ -26,8 +26,7 @@ <h2>Installing on Linux <a name='linux' href="#linux">#</a></h2>
 		<code>sudo yum install php81-php-xdebug3</code>
 	</li>
 	<li><strong>Debian</strong> (9/stretch, testing/buster/bullseye/sid):<br/><code>sudo apt-get install <a href='https://packages.debian.org/search?keywords=php-xdebug'>php-xdebug</a></code></li>
-	<li><strong>Fedora</strong> (32):<br/><code>sudo yum install php-xdebug</code></li>
-	<li><strong>Fedora</strong> (Remi Repo):<br/><code>sudo yum install php74-php-xdebug3</code></li>
+	<li><strong>Fedora</strong>:<br/><code>sudo dnf install <a href='https://packages.fedoraproject.org/pkgs/php-pecl-xdebug3/php-pecl-xdebug3/'>php-pecl-xdebug3</a></code></li>
```

Comment:
I don't think this is right. There is surely still a difference between Fedora itself, and Remi's Repo?

/cc @remicollet

Reply:
Sorry for not adding a comment regarding this.

`php-pecl-xdebug3` is (at the moment; I'm not sure since which version) part of the offical Fedora repositories (packaged by Remi) so I was unsure whether I should keep the mention of the custom repository or not.

Reply:
Xdebug is part of Fedora for years

The best command for any "foo" extension is "`dnf install php-foo`", so "`dnf install php-xdebug`", users don't have to worry about the package name. It was php-pecl-xdebug in the past (v2), it is php-pecl-xdebug3, for now, and maybe php-pecl-xdebug4 is the future (or something else, as "pecl" may disappear)

The command is the same with my repo for standard installation of default / single versions.
The "php74-php-xdebug" case is for multiple versions installation, not the common case, so it can probably be **removed** (and is explained by the [Wizard](https://rpms.remirepo.net/wizard/) to the repo user)

And the same can be applied to the RHEL and clones case.